tests/integration_tests/modules/test_user_events.py

```diff
@@ -77,7 +75,7 @@ def _test_network_config_applied_on_reboot(client: IntegrationInstance):
     assert 'dummy0' not in client.execute('ls /sys/class/net')
 
     _add_dummy_bridge_to_netplan(client)
-    client.execute('rm /var/log/cloud-init.log')
+    client.execute('echo "" > /var/log/cloud-init.log')
```

Comment:
Why do we need to touch /var/log/cloud-init.log in this test?

Reply:
Also still hitting:
2021-09-17 19:48:09 INFO      pycloudlib.instance:instance.py:167 executing: sudo -- sh -c 'cat /var/log/cloud-init.log'
FAILED                                                                   [ 66%]